src/nvim/main.c

```diff
@@ -1182,12 +1182,19 @@ static char_u *get_fname(mparm_T *parmp)
    * Expand wildcards in file names.
    */
   if (!parmp->literal) {
-    /* Temporarily add '(' and ')' to 'isfname'.  These are valid
-     * filename characters but are excluded from 'isfname' to make
-     * "gf" work on a file name in parenthesis (e.g.: see vim.h). */
+    cwd = xmalloc(MAXPATHL);
```

Comment:
`cwd` is not defined in this function. Perhaps move the allocation to main, and pass cwd to getfname?